delphi-checks/src/main/resources/org/sonar/l10n/delphi/rules/community-delphi/FullyQualifiedImport.html

```diff
@@ -0,0 +1,32 @@
+<h2>Why is this an issue?</h2>
+<p>
+    Imports should be done using the full name of the unit, as this facilitates code readability, avoids potential ambiguities about which unit is being imported, and ensures consistency throughout the codebase.
+</p>
+<p>
+    Using the full name of the units not only improves code clarity and maintenance but also optimizes the compilation process. The compiler does not need to spend time searching implicit scopes to resolve which units are being referenced. By specifying the full unit name, the compiler can immediately locate the necessary files, resulting in faster and more efficient compilation.
+</p>
+<h2>How to fix it</h2>
+<p>
+    Fix the import to use the full name of the unit.
+</p>
+<h3>Non-compliant Code Example</h3>
```

Comment:
We don't tend to use the "Non-compliant code example" headers, since the diff view of the code is usually explanatory enough. Would you be able to remove them?

Reply:
Of course! Done